modules/riseBidAdapter.js

```diff
@@ -316,7 +316,7 @@ function generateBidParameters(bid, bidderRequest) {
   };
 
   const pos = deepAccess(bid, `mediaTypes.${mediaType}.pos`);
-  if (pos) {
+  if (pos || pos === 0) {
```

Comment:
Do we need to add the same treatment to the other adapters like `kueezBidAdapter.js`, `minutemediaBidAdapter.js`, `publirBidAdapter.js`, `shinezBidAdapter.js`, `stnBidAdapter.js`?

Reply:
all adapters that are similar to our will need to have this treatment, not sure which of them.
@OronW @innay

Reply:
Yes. I would advise to check also `kueezBidAdapter`, `minutemediaBidAdapter`, `shinezBidAdapter` and `stnBidAdapter`.
 
I'm not familiar with (and couldn't find) `publirBidAdapter`, but if there any other adapters we own they would need to be updated as well